service/src/main/java/uk/gov/hmcts/reform/fpl/utils/EmailNotificationHelper.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.hmcts.reform.fpl.utils;
+
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.Respondent;
+import uk.gov.hmcts.reform.fpl.model.RespondentParty;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+
+import java.util.Objects;
+
+import static org.apache.commons.lang3.ObjectUtils.defaultIfNull;
+import static org.apache.commons.lang3.StringUtils.isNoneBlank;
+import static org.springframework.util.CollectionUtils.isEmpty;
+
+public class EmailNotificationHelper {
+
+    private EmailNotificationHelper() {
+    }
+
+    public static String buildSubjectLine(final CaseData caseData) {
+        final String lastName = getRespondents1Lastname(caseData);
+        final String familyManCaseNumber = defaultIfNull(caseData.getFamilyManCaseNumber(), "");
+
+        return String.format("%1$s%2$s%3$s", lastName,
+            (isNoneBlank(lastName, familyManCaseNumber) ? ", " : ""),
+            familyManCaseNumber);
+    }
+
+    private static String getRespondents1Lastname(final CaseData caseData) {
+        return isEmpty(caseData.getRespondents1()) ? "" : caseData.getRespondents1()
```

Comment:
this still isn't quite right. You don't need default if null up above as your getRespondentLast name returns either the name or "". i.e    

```
final String familyManCaseNumber = caseData.getFamilyManCaseNumber();
```

Reply:
we do need the `defaultIfNull` for `familyManCaseNumber` as it will return a `null` if not initialised - and we want an empty string when `null`. 

Also, `isEmpty(caseData.getRespondents1())` checks if the list is empty or `null`